.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: ruby
+
+script:
+- bundle exec jekyll build
+- tar czf website.tar.gz -C _site/ .
+
+deploy:
+  - provider: s3
+    access_key_id: $AWS_ACCESS_KEY
+    secret_access_key: $AWS_SECRET_KEY
+    bucket: $AWS_BUCKET
+    local_dir: ./_site
+    skip_cleanup: true
+    on:
+      branch: master
```

Comment:
Should we consider a super safe protected branch like "production" so we can merge to master at-will and then be intentional on when we do releases?

We don't operate like that now (we just immediately push content live). So, I don't think we need it, but just wanted to ask and acknowledge. Maybe in the future..

Reply:
I would suggest, for now, that we not.  This can change in the future as need warrants by adjusting the deployment conditions.

So far, a PR has covered just about everything that's been needed to change, and the change has needed to surface "immediately".